src/lib/mina_generators/zkapp_command_generators.mli

```diff
@@ -66,6 +66,7 @@ val gen_zkapp_command_from :
   -> ledger:Mina_ledger.Ledger.t
   -> ?protocol_state_view:Zkapp_precondition.Protocol_state.View.t
   -> ?vk:(Side_loaded_verification_key.t, State_hash.t) With_hash.Stable.V1.t
+  -> ?available_public_keys:unit Signature_lib.Public_key.Compressed.Table.t
```

Comment:
should this be a `Hash_set.t`, since the table is only mapping to `unit`?

Reply:
Maybe, I just used the type that was previously used in the generator. I can replace this type both here and in generator though.

Reply:
I might have been the one who used that type originally :-)

Yes, it would be better to use `Hash_set`

Reply:
Addressed